src/main/java/org/gitlab4j/api/models/Project.java

```diff
@@ -542,19 +544,44 @@ public void setStarCount(Integer starCount) {
         this.starCount = starCount;
     }
 
+    /**
+     * Tags will be removed in API v5
+     */
+    @Deprecated
     public List<String> getTagList() {
         return tagList;
     }
 
+    /**
+     * Tags will be removed in API v5
+     */
+    @Deprecated
     public void setTagList(List<String> tagList) {
         this.tagList = tagList;
     }
 
+    /**
+     * Tags will be removed in API v5
+     */
+    @Deprecated
     public Project withTagList(List<String> tagList) {
         this.tagList = tagList;
         return (this);
     }
 
+    public List<String> getTopics() {
+        return topics;
+    }
+
+    public void setTopics(List<String> topics) {
+        this.topics = topics;
+    }
+
+    public Project withTopics(List<String> topics) {
+        this.topics = topics;
+        return (this);
+    }
+
```

Comment:
This is interesting, but to use it, we need something similar to what exists for the old `TagList` in `ProjectApi.java`:

```
            if (project.getTagList() != null && !project.getTagList().isEmpty()) {
                formData.withParam("tag_list", String.join(",", project.getTagList()));
            }
 ```

Reply:
In `model/Project.java` I have used the same structure as in `withTagList`.

In `ProjectApi.java` i added TopicList to  ` updateProject(Project project)` and `createProject(Project project, String importUrl)` in the format you copied. 

i don't see the difference you are referring to.

Reply:
Yes this was done in 1cb36853bd56316acf99b6b7fbf9f48e4c094a5a after my initial comment